data/pu.txt

```diff
@@ -1,1515 +1,1344 @@
-- Recover 
-- Rapid Spin
+Floatzel @ Choice Band
+Ability: Water Veil
+EVs: 252 Atk / 4 Def / 252 Spe
+Jolly Nature
+- Waterfall
+- Ice Punch
+- Aqua Jet
+- Return
 
-Cryogonal @ Leftovers 
-Ability: Levitate 
-EVs: 248 HP / 156 Def / 104 SpD
-Calm Nature 
-- Freeze-Dry 
-- Rapid Spin
-- Recover 
-- Hidden Power [Fighting]
+Floatzel @ Life Orb
+Ability: Water Veil
+EVs: 252 Atk / 4 Def / 252 Spe
+Naive Nature
+- Waterfall
+- Ice Punch
+- Aqua Jet
+- Return / Hidden Power [Electric] / Ice Beam
 
+Floatzel @ Life Orb
+Ability: Water Veil
+EVs: 4 Atk / 252 SpA / 252 Spe
+Naive Nature
+- Hydro Pump
+- Ice Beam
+- Hidden Power [Electric] / Focus Blast
+- Aqua Jet
 
-=== [pu] Glalie ===
- 
-Glalie @ Focus Sash  
-Ability: Inner Focus  
-EVs: 252 Atk / 4 SpA / 252 Spe  
-Naive Nature  
-- Spikes  
-- Taunt  
-- Ice Beam / Freeze Dry
-- Explosion
+=== [Monferno] ===
 
-=== [pu] Camerupt ===
- 
-Camerupt @ Leftovers  
-Ability: Solid Rock  
-EVs: 252 HP / 4 SpA / 252 SpD 
-Timid Nature  
-- Stealth Rock  
-- Lava Plume  / Fire Blast
-- Earth Power  
-- Roar / Toxic
+Monferno @ Eviolite
+Ability: Iron Fist
+EVs: 252 Atk / 4 Def / 252 Spe
+Jolly Nature
+- Flare Blitz
+- Close Combat
+- Mach Punch
+- U-turn / Thunder Punch
 
-=== [pu] Altaria ===
- 
-Altaria @ Leftovers  
-Ability: Natural Cure  
-EVs: 116 HP / 252 Atk / 140 Spe  
-Jolly Nature  
-- Dragon Dance  
-- Dragon Claw  
-- Substitute / Earthquake
-- Roost
+Monferno @ Eviolite
+Ability: Iron Fist
+EVs: 252 Atk / 4 Def / 252 Spe
+Adamant Nature
+- Flare Blitz
+- Close Combat
+- Mach Punch
+- U-turn / Thunder Punch
 
-Altaria @ Life Orb  
-Ability: Natural Cure  
-EVs: 116 HP / 252 SpA / 140 Spe  
-Modest Nature  
-- Draco Meteor  
-- Roost  
-- Fire Blast  
-- Hidden Power [Fighting] / Toxic
+Monferno @ Eviolite
+Ability: Iron Fist
+EVs: 252 Atk / 4 Def / 252 Spe
+Adamant Nature
+- Swords Dance
+- Close Combat
+- Flare Blitz
+- Mach Punch
 
-=== [pu] Audino ===
- 
-Audino @ Leftovers  
-Ability: Regenerator  
-EVs: 252 HP / 4 Def / 252 SpD  
-Careful Nature  
-- Knock Off  
-- Wish  
-- Protect  
-- Encore / Toxic / Double Edge / Thunder Wave
+Monferno @ Life Orb
+Ability: Iron Fist
+EVs: 252 Atk / 4 Def / 252 Spe
+Adamant Nature
+- Swords Dance
+- Close Combat
+- Fire Punch
+- Mach Punch
 
-=== [pu] Bouffalant ===
- 
-Bouffalant @ Leftovers  
-Ability: Sap Sipper  
-EVs: 252 HP / 252 Atk / 4 SpD  
-Adamant Nature  
-- Substitute  
+=== [Leafeon] ===
+
+Leafeon @ Yache Berry  
+Ability: Chlorophyll
+EVs: 252 Atk / 4 SpD / 252 Spe  
+Jolly Nature  
 - Swords Dance  
-- Return  
-- Earthquake
- 
-Bouffalant @ Assault Vest
-Ability: Sap Sipper  
-EVs: 252 HP / 252 Atk / 4 SpD  
-Adamant Nature  
+- Leaf Blade  
+- Knock Off  
 - Return
+
+Leafeon @ Choice Band
+Ability: Leaf Guard
+EVs: 252 Atk / 4 Def / 252 Spe
+Jolly Nature
+- Bullet Seed
+- Leaf Blade
+- Knock Off
+- X-Scissor / Baton Pass / Quick Attack
+
+=== [Mr. Mime] ===
+
+Mr. Mime @ Choice Scarf
+Ability: Soundproof / Filter

```

Comment:
i am also terrible with github so i can't rebase and edit the current pull request etc so yeah